addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onkyo" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Commands -->
+    <channel-type id="power">
+        <item-type>Switch</item-type>
+        <label>Power</label>
+        <description>Power on/off your device</description>
+    </channel-type>
+    <channel-type id="input">
```

Comment:
Done. As all zone groups share the same channel types, I'm not able to move zone 2 and 3 to advanced, which could be best option as most of the users probably use only zone 1. Zone groups doesn't support advanced flag either. Do you have any tricks on mind?